Tests/test_cellosaurus.py

```diff
@@ -22,17 +22,17 @@ def test_read(self):
         self.assertEqual(record["DR"][1], ("ECACC", "94050311"))
         self.assertEqual(record["DR"][2], ("IHW", "IHW9326"))
         self.assertEqual(record["DR"][3], ("IMGT/HLA", "10074"))
-        self.assertEqual(record["WW"][0], "http://bioinformatics."
-                                          "hsanmartino.it"
-                                          "/ecbr/cl326.html")
-        self.assertEqual(record["CC"][0],
-                         "Part of: 12th International "
-                         "Histocompatibility Workshop "
-                         "(12IHW) cell line panel.")
-        self.assertEqual(record["CC"][1],
-                         "Transformant: EBV.")
-        self.assertEqual(record["OX"][0],
-                         "NCBI_TaxID=9606; ! Homo sapiens")
+        self.assertEqual(
+            record["WW"][0], "http://bioinformatics.hsanmartino.it/ecbr/cl326.html",
+        )
+        self.assertEqual(
+            record["CC"][0],
+            "Part of: 12th International "
+            "Histocompatibility Workshop "
+            "(12IHW) cell line panel.",
```

Comment:
Can you get that implicitly concatenated string on one line?

Reply:
2 lines, one line exceeds; done.